### PR TITLE
[JBPM-9436] AMQ Streams (Kafka) listener functionality / consumer

### PR DESCRIPTION
**Pull request by fjtirado**
Adding Kafka test classifier

**JIRA**:  

[JBPM-9436](https://issues.redhat.com/browse/JBPM-9436)

Merge with

- https://github.com/kiegroup/droolsjbpm-integration/pull/2288
- https://github.com/kiegroup/jbpm/pull/1785

### CODE REVIEW

**Comment by mareknovotny** [comment]
jenkins do fdb

**Review by afalhambra — COMMENTED** [review]
Just a comment - you can skip it if you want, not an issue, just out of curiosity.
Looks good to me.

File: pom.xml [hunk 1]
```diff
@@ -5370,7 +5370,12 @@
         <artifactId>kafka-clients</artifactId>
         <version>${version.org.apache.kafka}</version>
       </dependency>
-
+      <dependency>
+        <groupId>org.apache.kafka</groupId>
+        <artifactId>kafka-clients</artifactId>
+        <version>${version.org.apache.kafka}</version>
+        <classifier>test</classifier>
+      </dependency>
```
[the remark this answers is not in the document]
Comment (fjtirado) [reply pom.xml:5378]: What is removed from drools-ha is the managed dependeny, but the real dependency is still there in the children. So, this is done to centralize all kafka versiones in just one place, kie-parent

**Review by afalhambra — APPROVED** [review]
Nice work!

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
Please always add into top level PR description all dependent PRs.

**Pull request merged**